lib/registration.pm

```diff
@@ -223,7 +224,7 @@ sub ssh_add_suseconnect_product {
 
     remove_suseconnect_product($name, [$version, [$arch, [$params]]]);
 
-Wrapper for SUSEConnect -d $name.
+wrapper for suseconnect -d $name.
```

Comment:
what the purpose of this change ?

Reply:
There is no purpose, it's a mistake caused by my new keyboard. I'm sorry